<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-4J1ULF8" failures="1" tests="5" name="div tests" time="0.009" errors="0" timestamp="2021-03-26T12:55:53 MSK">
  <testcase classname="hw1.arithmetic.DivOperationTest" name="divByZeroForDoubleValuesTest" time="0.002">
    <failure type="java.lang.AssertionError" message="Expected Throwable to be thrown, but nothing was thrown">
      <![CDATA[java.lang.AssertionError: Expected Throwable to be thrown, but nothing was thrown
at hw1.arithmetic.DivOperationTest.divByZeroForDoubleValuesTest(DivOperationTest.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- divByZeroForDoubleValuesTest -->
  <testcase classname="hw1.arithmetic.DivOperationTest" name="divByZeroForLongValuesTest" time="0.0"/>
  <testcase classname="hw1.arithmetic.DivOperationTest" name="divDoubleValuesTest" time="0.001"/>
  <testcase classname="hw1.arithmetic.DivOperationTest" name="divDoubleValuesTest" time="0.001"/>
  <testcase classname="hw1.arithmetic.DivOperationTest" name="divLongValuesTest" time="0.001"/>
</testsuite> <!-- div tests -->
